### ISSUE TITLE
Fields: Tooltips - Stage 1 - Add support of tooltips to OfField 

### ISSUE THREAD

**Issue by alexivanenko** [1CRM/oceanfront#454]
It should be a 'title' attribute becaue it's not possible to use it on devices with touch screen. 
It should be a new optional property in OfField. Pl;ease adjust styles for different field types.

This is the design from @AlexLukasevych for the admin area, you can find a tooltip example here - https://www.figma.com/design/InnSYDWOJxw2lygbRBGxQp/Admin%3A-Tooltips?node-id=24-2037

 <img width="1217" alt="Снимок экрана 2024-11-12 в 16 43 26" src="https://github.com/user-attachments/assets/a9557748-cbb0-4d1b-938e-8a8cfc0aae3c">

@ivan-b implemented tooltips in the HTML Editor, you can see how it's done. 
<img width="910" alt="Снимок экрана 2024-11-12 в 16 47 24" src="https://github.com/user-attachments/assets/33b17ae1-70ff-4e66-8e06-9f58cbcb7781">

**Comment by ademenev** [comment]
@ivan-b I recently looked at some UI libraries and many of them have issues with tooltips position when the trigger element is close to the page edge, like the inputs on the screenshot. We should handle this properly and reposition the tooltip if there is not enough space to display it in the default place

  ![Image](https://github.com/user-attachments/assets/e82020f0-56b2-45b0-a611-4fbde042efda)

**Comment by ivan-b** [comment]
@ademenev yes, I've handle such a situation, the tooltip will change the side if there is not enough space and if there is more space on the other side
